tools/java/org/unicode/cldr/test/ConsoleCheckCLDR.java

```diff
@@ -1068,7 +1068,7 @@ private static void addDataToErrorFile(String localeID, String path, ErrorType s
             Subtype subType) {
             String section = path == null
```

Comment:
We could adapt the API so it handled "null" and returned "general" as a default, but I'm not even sure if this code is being used.